src/main.cpp

```diff
@@ -47,7 +47,7 @@
 #include "util/backports.h"
 #include "util/recording_translator.h"
 
-int main(int argc, char** argv)
+int AppMain(int argc, char** argv)
 {
```

Comment:
"AppMain" is not an acceptable name for a free function. It must start with a lower case letter.